.gitignore

```diff
@@ -130,3 +130,6 @@ dmypy.json
 
 # IDE
 .idea/
+
+# Auto scanner file
+.autoscanner-state-file
```

Comment:
I thought about this but thought maybe since it was user configurable we wouldn't do it, but this makes sense since it's a pretty advanced config parameter.

Reply:
Totally, for the default I think it makes sense to add. If they're going to pick a different path, it'd likely be out of the repo scope anyway I think.